cmd/zpool/Makefile.am

```diff
@@ -1,5 +1,7 @@
 include $(top_srcdir)/config/Rules.am
 
+AM_CFLAGS += -Wno-stringop-truncation
```

Comment:
maybe make this dependent on debug build?

Reply:
its not critical. as i said it was disabled by default on older compilers and will just create false positives if strncpy is used. if you remove it on release builds you still get a annoying warning. its just no error then. in fact it would be a better way to change the sourcecode to deal with the warning to make the compiler happy. but i dont think this is neccessary

Reply:
but i had to exclude it for freebsd which uses clang for compiling. basicly i need to find a way to check for clang and also older compiler versions which do not support this flag. i believe it was introduced with gcc 7

Reply:
yeah that's ok but I still can't see where those errors happen in the tests.. am I blind? :D